zebra-rpc/src/methods/tests/prop.rs

```diff
@@ -20,7 +21,8 @@ proptest! {
 
         runtime.block_on(async move {
             let mut mempool = MockService::build().for_prop_tests();
-            let rpc = RpcImpl::new("RPC test".to_owned(), Buffer::new(mempool.clone(), 1));
+            let state = zebra_state::init_test(Network::Mainnet);
```

Comment:
Optional: since the state service isn't actually used in these tests, we could use `MockService` instances instead, and then check at the end of the test that no requests were sent to it.

Reply:
Good idea, i think i applied everywhere in https://github.com/ZcashFoundation/zebra/pull/3707/commits/6a4055f2030fb896b89f9c015edb6aa0c97be606